ChatQnA/tests/test_manifest_on_gaudi.sh

```diff
@@ -37,11 +37,12 @@ function install_chatqna {
 
 function get_end_point() {
     # $1 is service name, $2 is namespace
-    ip_address=$(kubectl get svc $1 -n $2 -o jsonpath='{.spec.clusterIP}')
-    port=$(kubectl get svc $1 -n $2 -o jsonpath='{.spec.ports[0].port}')
+    ip_address=$(kubectl get nodes -o jsonpath='{.items[0].status.addresses[?(@.type=="ExternalIP")].address}')
+    port=$(kubectl get svc $1 -n $2 -o jsonpath='{.spec.ports[0].nodePort}')
```

Comment:
nope, not all K8S services have nodePort type

Reply:
Tests should go through Nginx before testing the backend to ensure the Nginx configuration is validated. This repository will be closed due to the number of example manifests; I will submit each as a separate pull request.